models/_sector/nft/trades/ethereum/platforms/foundation_ethereum_base_trades.sql

```diff
@@ -0,0 +1,127 @@
+    , seller
+    , buyer
+    , creatorRev+totalFees+sellerRev AS price_raw
+    , contract_address AS project_contract_address
+    , nftContract AS nft_contract_address
+    , evt_tx_hash AS tx_hash
+    , totalFees AS platform_fee_amount_raw
+    , case when (sellerRev = 0 and cast(creatorRev as decimal(38)) > 0 ) then 0 else creatorRev end AS royalty_fee_amount_raw
+    , evt_index as sub_tx_trade_id
+    FROM {{ source('foundation_ethereum','market_evt_BuyPriceAccepted') }} f
+     {% if is_incremental() %} -- this filter will only be applied on an incremental run
+     WHERE f.evt_block_time >= date_trunc("day", now() - interval '1 week')
+     {% else %}
+     WHERE f.evt_block_time >= '{{project_start_date}}'
+     {% endif %}
+    UNION ALL
+    SELECT
+     evt_block_time AS block_time
+    , evt_block_number AS block_number
+    , tokenId AS nft_token_id
+    , 'Sell' AS trade_category
+    , case when (sellerRev = 0 and cast(creatorRev as decimal(38)) > 0 ) then 'primary' else 'secondary' end as trade_type
+    , seller
+    , buyer
+    , creatorRev+totalFees+sellerRev AS price_raw
+    , contract_address AS project_contract_address
+    , nftContract AS nft_contract_address
+    , evt_tx_hash AS tx_hash
+    , totalFees AS platform_fee_amount_raw
+    , case when (sellerRev = 0 and cast(creatorRev as decimal(38)) > 0 ) then 0 else creatorRev end AS royalty_fee_amount_raw
+    , evt_index as sub_tx_trade_id
+    FROM {{ source('foundation_ethereum','market_evt_OfferAccepted') }} f
+     {% if is_incremental() %} -- this filter will only be applied on an incremental run
+     WHERE f.evt_block_time >= date_trunc("day", now() - interval '1 week')
+     {% else %}
+     WHERE f.evt_block_time >= '{{project_start_date}}'
+     {% endif %}
+    UNION ALL
+    SELECT
+     evt_block_time AS block_time
+    , evt_block_number AS block_number
+    , tokenId AS nft_token_id
+    , 'Private Sale' AS trade_category
+    , case when (sellerRev = 0 and cast(creatorFee as decimal(38)) > 0 ) then 'primary' else 'secondary' end as trade_type
+    , seller
+    , buyer
+    , creatorFee+protocolFee+sellerRev AS price_raw
+    , contract_address AS project_contract_address
+    , nftContract AS nft_contract_address
+    , evt_tx_hash AS tx_hash
+    , protocolFee AS platform_fee_amount_raw
+    , case when (sellerRev = 0 and cast(creatorFee as decimal(38)) > 0 ) then 0 else creatorFee end AS royalty_fee_amount_raw
+    , evt_index as sub_tx_trade_id
+    FROM {{ source('foundation_ethereum','market_evt_PrivateSaleFinalized') }} f
+     {% if is_incremental() %} -- this filter will only be applied on an incremental run
+     WHERE f.evt_block_time >= date_trunc("day", now() - interval '1 week')
+     {% else %}
+     WHERE f.evt_block_time >= '{{project_start_date}}'
+     {% endif %}
+    )
+
+SELECT
+  'ethereum' as blockchain
+, 'foundation' as project
+, 'v1' as version
```

Comment:
a minor thing that i've wondered awhile: `nft.trades` uses `v` prefix on versions, yet `dex.trades` does not -- should we standardize and remove the `v`?

Reply:
Always a fan of standardization, but I think I might prefer `v` prefixed in this case as it forces the column to be interpreted as a string.

Reply:
fair, as some `dex.trades` values are actual strings